tests/sdk/python/get-one-step/__main__.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016-2020, Pulumi Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pulumi
+from pulumi_kubernetes.apiextensions.CustomResource import CustomResource
+from pulumi_kubernetes.apiextensions.v1beta1.CustomResourceDefinition import (
```

Comment:
apiextensions/v1beta1 is deprecated and will be removed in the next k8s release. It would be better to use apiextensions/v1 for these.

Reply:
SG